core/src/components/checkbox/checkbox.ionic.scss

```diff
@@ -1,34 +1,46 @@
-@import "./checkbox";
```

Comment:
Please revert this. We want to be consistent, we import the base styles to all components for all themes. If we want to avoid values being mixed, then we would use the variables to override any values as shown in other themes.

Reply:
See my answer [here](https://github.com/ionic-team/ionic-framework/pull/29561#discussion_r1622030340), please.

We can leave this PR in stand by for now :)